duckduckgo/sync_crypto
# Windows: Update libsodium to 1.0.19 and include ARM64 binaries

Task/Issue URL: https://app.asana.com/0/1201048563534612/1206705899006123/f

### Description

Package and publish an ARM64 version of the Nuget package. To achieve this, update libsodium to version [1.0.19](https://github.com/jedisct1/libsodium/releases/tag/1.0.19-RELEASE) (v1.0.18 is currently used). 

#### Additionally:

Add the ability to publish a prerelease Nuget package version. Prerelease versions contain `-something` postfix.

### Step to test this PR
1. Checkout this branch.
2. In `windows` directory execute `.\publish.ps1 1.0.5.0 test` command. This should publish x64, x86 and ARM64 Nuget packages with version `1.0.50-test`.
3. In the `windows-browser` repo checkout `spike/alex/arm` branch.
4. Change the `WindowsBrowser.csproj` to use `1.0.50-test` version of the Sync Crypto packages: https://github.com/duckduckgo/windows-browser/blob/refs/heads/spike/alex/arm/WindowsBrowser/WindowsBrowser.csproj#L264.
5. Check that Sync works in x64 and x86 Debug and Release builds.
6. Optionally check that Sync works in ARM Release build by committing changes from step 4 and building a product review build from that commit. You'll need a VM with ARM Windows.
